### ISSUE TITLE
On saving HTML box

### ISSUE THREAD

**Issue by EdwardBock** [palasthotel/grid#376]
@mkernel 
I get an error when I try to save a html box:
```
Method: UpdateBox
Status: (200)  parsererror
ResponseText: Fatal error:  Cannot use object of type grid_html_box as array in /Users/edward/Projekte/Palasthotel/GRID/grid-wordpress.local/wp-content/plugins/grid/wordpress_plugin.php on line 126
```
I'm not really shure whats happening there and what the applied filter does. Can you have a look on our next grid day?

**Comment by mkernel** [comment]
That's a severe bug! Please package 1.3.2! See grid-wordpress Issue 48

**Comment by EdwardBock** [comment]
fixed with #378
